conjureup/ui/widgets/step.py

```diff
@@ -175,6 +176,19 @@ def generate_additional_input(self):
         self.step_pile.focus_position = self.current_button_index
 
     def submit(self, btn):
+        for i in self.additional_input:
+            if i['input'] and (
+                    i['input'].value is None and self.model.required):
+                self.app.log.debug("Missing default: {}".format(i))
```

Comment:
this log message is a little confusing, why 'missing default'?
maybe 'missing required input:' ?